### PR TITLE
Update Polish translation

### PR DESCRIPTION
**Pull request by Arthusppp**
Update Polish translation

### CODE REVIEW

File: files/lang/pl.po [hunk 1]
```diff
@@ -2214,13 +2214,13 @@ msgstr ""
 "proponuje ci pomoc w rozwinięciu wybranej przez ciebie umiejętności."
 
 msgid "You can't afford to upgrade your troops!"
-msgstr ""
+msgstr "Nie możesz pozwolić sobie na ulepszenie oddziałów!"
```
Comment (michkry) [inline files/lang/pl.po:2217]: It's correct but we may also consider translating it like this: "Nie stać cię na ulepszenie oddziałów!"

Comment (Arthusppp) [reply files/lang/pl.po:2217]: Totally agree! Thanks!

**Comment by Arthusppp** [comment]
Hi @ihhub, this PR has been reviewed and it is ready for merge! :)

**Comment by ihhub** [comment]
@Arthusppp , thanks a lot for helping with Polish translation!

**Pull request merged**